documentation/faust/setup.md

```diff
@@ -0,0 +1,136 @@
+# Setting up the Environment
+
+## Requirements
+
+Make sure you have the following requirements before starting:
+
+- A [GitHub account](https://github.com/join)
+- [`git`](https://git-scm.com/download)
+- [Python 3](https://www.python.org/downloads/)
+- [Homebrew](https://brew.sh) on macOS X
+- [Xcode](https://developer.apple.com/xcode/) on macOS X
+- [FAUST](https://faust.grame.fr/downloads/)
+
+You then only need minimum knowledge on how the terminal works to get going.
+
+
+## Setting up FAUST
+
+FAUST comes with some programs, but also its headers, and it should be configured so
+that FAUST headers are available to the development environment.
+
+In general it is best to install it with your package manager. This will make sure that both
+the FAUST programs and development environment are correctly installed.
+
+This is typically done like this on macOS:
+
+```
+brew install faust
+```
+
+or on Linux:
+
+```
+sudo apt-get install faust
+```
+
+
+## Cloning
+
+Then, clone the repository. This will create a `eurorack-blocks` directory **into the current working directory**, with all dependencies.
+
+
+```shell-session
+~$ git clone --recurse-submodules git@github.com:ohmtech-rdi/eurorack-blocks.git
+~$ cd eurorack-blocks
+~/eurorack-blocks$
+```
+
+
+## Setting up
+
+Install by running the script below:
+
+```shell-session
+~/eurorack-blocks$ ./build-system/install.py
+```
+
+This will add the `erbb` script to your `PATH`.
```

Comment:
Ah damn you are right, and I think I have the same mistake in the C++ setup guide.
Normally sourcing it should be enough, but I'll keep both methods I think:

```suggestion
This will add the `erbb` script to your `PATH`.
Be sure to reload your shell configuration after this, for example running `source ~/.bash_profile` when using `bash`.
Alternatively, you might start a new terminal to use the new shell configuration.
```

Reply:
OK